internal/backends/postgresql/metadata/metadata.go

```diff
@@ -149,10 +152,15 @@ func (c *Collection) unmarshal(doc *types.Document) error {
 	if v, _ := doc.Get("cappedSize"); v != nil {
 		c.CappedSize = v.(int64)
 	}
+
```

Comment:
If we add an empty line there, then we should have an empty line after the comment above, so it belongs to all following code and not to a single block